app/javascript/styles/mastodon/components.scss

```
@@ -5205,6 +5204,7 @@ a.status-card {
     color: $darker-text-color;
     cursor: default;
     pointer-events: none;
+    margin-left: 16px - 2px;
```

This breaks right-to-left layouts, I think the following should fix it:
```suggestion
    margin-inline-start: 16px - 2px;
```

---

Good point. 👍